.github/workflows/pack-debian.yml

```diff
@@ -166,6 +165,8 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@v4
+      - name: Check the tag consistency with the source version
```

Comment:
Check this in the `build` job if the workflow is launched on a Git tag (see above)?

Reply:
Done as above.